pkg/render/manager.go

```diff
@@ -107,8 +107,8 @@ func Manager(cfg *ManagerConfiguration) (Component, error) {
 		}
 	}
 
+	tlsAnnotations[cfg.InternalTrafficSecret.HashAnnotationKey()] = cfg.InternalTrafficSecret.HashAnnotationValue()
```

Comment:
This should be guarded with a nil-check for managed clusters if the other comment is implemented.